src/OpenAI.csproj

```diff
@@ -48,4 +48,10 @@
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
     <PackageReference Include="System.Text.Json" Version="8.0.2" />
   </ItemGroup>
+
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
Consider using the [`<InternalsVisibleTo>`](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#internalsvisibleto) item, introduced in dotnet 5.